config/config.go

```diff
@@ -265,6 +265,27 @@ type DeploymentConfig struct {
 // defaults to the namespace configured for IstioNamespace (which itself defaults to 'istio-system').
 type IstioComponentNamespaces map[string]string
 
+
+// Threshold configuration
+type Threshold struct {
+	Namespace string `yaml:"namespace"`
+	ThresholdAlerts    []ThresholdAlert  `yaml:"threshold_alerts"`
+}
+
+type ThresholdAlert struct {
```

Comment:
ThresholdAlert -> ThresholdCondition or ThresholdProperties.

Alert is a complex term and using it here it may create amgibuity.